src/common/Makefile.am

```diff
@@ -103,10 +99,8 @@ libcommon_internal_la_SOURCES += \
 endif
 
 # used by RBD and FileStore
-if LINUX
```

Comment:
@rohanmars Sorry, my previous message (it is also used by FileStore) actually was for this comment.